src/Controller/UserController.php

```diff
@@ -109,20 +109,19 @@ public function deleteUser(Request $request, array $requestAttributes): Response
         $this->userService->delete($id);
         $body = Stream::createFromString('');
         $response = new Response($body, '1.1', 301, '');
-        $response = $response->withHeader('Location', 'http://local.quiz.com/listUsers');
+        $response = $response->withHeader('Location', 'http://local.quiz.com/listUsers?page=1');
 
         return $response;
     }
 
     public function getUsers(Request $request, array $requestAttributes): Response
     {
-        $userRepo = $this->repositoryManager->getRepository(User::class);
-        // get filters and pagination from request
-        $criteria = new Criteria();
-        $users = $userRepo->findBy($criteria);
-        $users = ['users' => $users];
+        $page = (int)$request->getParameter('page');
```

Comment:
You have a variable called $pages which is in fact the number of users that exists in the application. Pls change the variable name to something less confusing.